src/App.tsx

```diff
@@ -834,6 +836,15 @@ function App(): ReactElement {
                           >
                             Show scale bar
                           </Checkbox>
+                          <Checkbox
+                            type="checkbox"
+                            checked={showTimestamp}
+                            onChange={() => {
+                              setShowTimestamp(!showTimestamp);
+                            }}
+                          >
+                            Show timestamp
+                          </Checkbox>
```

Comment:
Should this checkbox be disabled if the dataset doesn't have the right metadata to show time?

Reply:
We *could* show frame numbers as the time overlay in that case, but it's probably not going to bother anyone if the checkbox is there either way.